.github/workflows/static-checks.yml

```diff
@@ -51,7 +51,16 @@ jobs:
         with:
           distribution: 'zulu'
           java-version: ${{ matrix.java }}
-          cache: 'maven'
+
+      - name: cache maven dependencies
+        run: ${MVN} dependency:resolve dependency:resolve-plugins
```

Comment:
could you  add `-B`  or `-q` to get rid of download progress

Reply:
it might make sense to add those to the ${MVN} alias directly

Reply:
@xvrl - https://stackoverflow.com/questions/58546741/excluding-some-dependency-from-mvn-dependencyresolve - This may work.

Reply:
let's not mix things in this thread. With respect to `-B` / `-q` we already have `-B` in the $MVN alias. I added `--no-transfer-progress` to address the problem in all our MVN aliases.